x-pack/solutions/observability/plugins/apm/public/components/app/mobile/service_overview/stats/metric_item.tsx

```diff
@@ -40,12 +40,12 @@ export function MetricItem({
           <EuiSkeletonText lines={3} />
         </EuiPanel>
       ) : (
-        <EuiErrorBoundary>
+        <KibanaErrorBoundary>
```

Comment:
Hi @jennypavlova,
Is this supposed to capture errors for a section of the page? If yes, [KibanaSectionErrorBoundary](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/shared-ux/error_boundary/README.mdx) is the option for it.

Reply:
As discussed, I added a PR to remove it - https://github.com/elastic/kibana/pull/227412,
@maryam-saeidi  Thank you for reviewing ❤️